src/main/java/net/glowstone/net/handler/play/player/PlayerAbilitiesHandler.java

```diff
@@ -15,7 +17,13 @@ public void handle(GlowSession session, PlayerAbilitiesMessage message) {
 
         GlowPlayer player = session.getPlayer();
         boolean flying = (message.getFlags() & 0x02) != 0;
```

Comment:
Looks good, however rename this local variable to `flyingFlag` to avoid confusion.